CHANGES_UNRELEASED.md

```diff
@@ -18,3 +18,13 @@ Use the template below to make assigning a version number during the release cut
   - Description of the change with a link to the pull request ([#0000](https://github.com/mozilla/application-services/pull/0000))
 
 -->
+
+## Tabs
+
+### What's Changed
+
+The Tabs engine no longer resets after every sync:
```

Comment:
nano-nit - "reset" probably doesn't mean much to consumers - maybe change this to describe it as making the tabs engine more efficient or something?